examples/web-wagmi/src/profiles/ProfilesPage.tsx

```diff
@@ -56,6 +56,11 @@ const profileHooks = [
     description: `Switch active profile.`,
     path: '/profiles/useActiveProfileSwitch',
   },
+  {
+    label: 'useProfilesWhoMirroredPublication',
+    description: 'Fetch profiles that have mirrored a specific publication',
+    path: '/discovery/useProfilesWhoMirroredPublication',
```

Comment:
wrong path
```suggestion
    path: '/profiles/useProfilesWhoMirroredPublication',
```

Reply:
or in fact I would move this hook to `publications` category and rename it 
```suggestion
    path: '/publications/useWhoMirroredPublication',
```